### PR TITLE
fix: Add what to about.py

### PR DESCRIPTION
**Pull request by sreyasaju**

### CODE REVIEW

File: sugaroid/brain/about.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def process(self, statement, additional_response_selection_parameters=None):
 
         emotion = Emotion.neutral
         if self.pronoun.lower().startswith("you"):
-            if self.quest.lower() == "who":
+            if self.quest.lower() == "who" or "what":
```
Comment (srevinsaju) [inline sugaroid/brain/about.py:89]: what?

**Review by srevinsaju — CHANGES_REQUESTED** [review]

File: sugaroid/brain/about.py (hunk 1 above)
Comment (srevinsaju) [reply sugaroid/brain/about.py:89]: Did you test your change?

Comment (sreyasaju) [reply sugaroid/brain/about.py:89]: Hmm.. but 'who is your hobby' won't be grammatically correct..
Well, I think it won't be appropriate to ask `"Who" is your favorite color?`but instead we would ask `What is your favorite color?`

Comment (srevinsaju) [reply sugaroid/brain/about.py:89]: I understood your change. I just asked if you tested your change.

**Review by srevinsaju — CHANGES_REQUESTED** [review]

File: sugaroid/brain/about.py (hunk 1 above)
Comment (sreyasaju) [reply sugaroid/brain/about.py:89]: Well, if  you think it is not needed, I will revert it.. :)

Comment (srevinsaju) [reply sugaroid/brain/about.py:89]: I understood your change. I just asked if you tested your change.

Comment (srevinsaju) [reply sugaroid/brain/about.py:89]: > Well, if you think it is not needed, I will revert it.. :)

It is needed. You still haven't answered all of my questions yet.

Comment (sreyasaju) [reply sugaroid/brain/about.py:89]: > I just asked if you tested your change.

hmm yea and I got this as the result:
```
( ဖ‿ဖ) @> what is your hobby
sugaroid: Calculating random binary sequences and chatting with you!

( ဖ‿ဖ) @> who is your hobby
sugaroid: Calculating random binary sequences and chatting with you!

( ဖ‿ဖ) @> who is your father 
sugaroid: Mr Charles Babbage?

( ဖ‿ဖ) @> what is your fathers name                       
sugaroid: Mr Charles Babbage?
```

Comment (srevinsaju) [reply sugaroid/brain/about.py:89]: Interesting. This is not a fix. This is now a hack.

```python
"Python 3.9.1 (default, Dec 13 2020, 11:55:53) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> "who" or "what"
'who'
>>> "who" or "what"
'who'
>>> "who" or "what"
'who'
>>> "who" or "what"
'who'
>>> "who" or "what"
'who'
>>> 
>>> "who" or "what"
'who'
>>> k = "who"
>>> k == "who" or "what"
True
>>> k == "who" or "what"
True
>>> k = "what"
>>> k == "who" or "what"
'what'
>>> k == "who" or "what"
'what'
>>> k = "wow"
>>> k == "who" or "what"
'what'
```

**Comment by srevinsaju** [comment]
Thanks. Merged by hand.

**Pull request closed without merging**